src/Tribe/Views/V2/iCalendar/Traits/Outlook_Methods.php

```diff
@@ -72,8 +72,10 @@ protected function generate_outlook_add_url_parameters( $calendar = 'live' ) {
 		} else {
 			$enddt = Dates::build_date_object( $event->end_date, $timezone )->format( 'c' );
 		}
+		$enddt = str_replace( [ '+', ':' ], [ '%2B', '%3A' ], $enddt );
```

Comment:
Would this replacement be better in a function that can be shared between the start & end? Could also have a filter just in case other replacements need to be done.

Reply:
For the snapshots, I have already ran that suite with --debug and committed the new snapshots. I will run it to all suites to see if there are more, thanks.

For that replacement, [in the ticket](https://stellarwp.atlassian.net/browse/TEC-5085) there was a suggested fix with a deprecated function via a filter. I investigated it to see the alternatives and figured out that since it is the result 0f `Dates::build_date_object` only those two characters need to be decode. That `str_replace` was the lighter, imo. [Here's the alternatives](https://php-play.dev/?c=DwfgDgFmBQAkAmBDALgUwAQF50HIBMADHgCwC0BAHKYQCoECcAXAQcwQNQEDMbOA3NGioAxhAD26BClQChoiTmAQATgD5%2BgkePQBXZQBtUAO2Fj4qABS7kAMwoB9c6fNWpadAEpPfdAHpf6AAiqGDKItLwAHSa8rhKahpy2sqIAO56hiZmlugAtgBG9qZGAG6oysj2xs4AlkYA5q5IaAA0uACqNABipBQ4bTgAkgDKAPK9FACs9KQAjDie3jHaiirqsloSAM7IyvZhYPqIwjk47P24AKR4AEIXO3sHRydWOIwXOJdcAIIXbhheDwbWKrBLA7QPfYhZ45ADauHOA3e6AAum14Z9bh8vr9UW1-kskgp4utlttdrsmtJ0QiFphVFcsUi6QzPj8FiilkA&v=8.3&f=html). What do you think?